src/bees.cc

```diff
@@ -56,6 +57,7 @@ do_cmd_help(char *argv[])
 		"\n"
 		"Workarounds:\n"
 		"    -a, --workaround-btrfs-send    Workaround for btrfs send\n"
+		"    -N, --no-tmpfile-compression   Disable compression for temporary files\n"
```

Comment:
I think the naming could be better: What it actually wants to do is decompressing shared extents. Nobody cares if this is done through tmpfiles or other means. Options should try not exposing too many details of the internal workings so they can still be true after code redesigns.